### ISSUE TITLE
Divide to zero

### ISSUE THREAD

**Issue by punzik** [cleverca22/toxvpn#8]
Error in the last commit:

```
$ sudo gdb ./toxvpn
(gdb) run
creating interface
what is the VPN ip of this computer?
192.168.200.2
my id is 259e71d4257e7c2eb21628698d491b70ebc907d63d5c7cff5e729ae469e00d41b4c532f18ae3 and IP is 192.168.200.2

Program received signal SIGFPE, Arithmetic exception.
0x000000000040c94c in ToxVPN::do_bootstrap (tox=0x62fd40, nodes=std::vector of length 0, capacity 0)
    at /srv/users/np/source/toxvpn/main.cpp:50
50	    size_t i = rand() % nodes.size();
(gdb) print(nodes.size())
$1 = 0
(gdb) print(nodes)
$2 = std::vector of length 0, capacity 0
```

ArchLinux, gcc (GCC) 6.2.1 20160830.

**Comment by cleverca22** [comment]
it needs to be compiled with -DBOOTSTRAP_FILE=/path/to/bootstrap.json, so it can find the bootstrap nodes at runtime

it looks like i need to add proper error handling for when that file cant be opened

**Comment by punzik** [comment]
I think path to bootstrap.json need to specify as command line option. Or look in hardcoded path e.g. /etc/toxvpn, ~/.toxvpn, $PREFIX/share/toxvpn.

**Comment by taktoa** [comment]
Fixed by 6f9b73885e3334b45e50ac9cca04e12d100c0be
